trunk/SUAVE/Components/Energy/Converters/Rotor.py

```diff
@@ -0,0 +1,798 @@
+## @ingroup Components-Energy-Converters
```

Comment:
The code in this file looks quite similar to the propeller. Can they be combined?

Reply:
They are only small differences as of now but I have a PR in the near future that will make them differ a bit. I think for now leaving the script in is beneficial. What happened was that Mike Vegh created the rotor script and added a few lines to handle hover.